### ISSUE TITLE
`selectText()` on TextEditor select wrong text when tabs are used for indention

### ISSUE THREAD

**Issue by openscript** [webdriverio-community/wdio-vscode-service#96]
If tabs are used in a file, where `selectText()` is executed, the cursor will stop before the text, which should be selected. The reason for this, could be that columns and coordinates don't correspond in VSCode:

https://github.com/webdriverio-community/wdio-vscode-service/blob/88c7a47d041b126e84ae1703d04bc13d7648e4bf/src/pageobjects/editor/TextEditor.ts#L344C44-L344C50

If the tab size is two moving one tab to the right increases the column count by two.

**Comment by seanpoulter** [comment]
I don't think we know the right way to solve this @openscript. What are our options? You've already suggested one. I can think of another option to increment by the size of a tab if we find one. We should be able to check the [`TextEditorOptions`](https://code.visualstudio.com/api/references/vscode-api#TextEditorOptions).

I'm also curious what test cases we'd want to cover. I'm fairly confidence searching for whitespace at the end of a sentence won't work as expected due to a `.trim()`. :thinking: 

* Spaces
* Tabs (size 4)
* A mix of spaces and tabs
* Searching for text with trailing spaces at the end of the line (:bug:?)
